subprojects/gradle/lint-report/src/integTest/resources/lint-results.xml

```diff
@@ -9,12 +9,12 @@
         priority="6"
         summary="Implied default locale in case conversion"
         explanation="Calling `String#toLowerCase()` or `#toUpperCase()` **without specifying an explicit locale** is a common source of bugs. The reason for that is that those methods will use the current locale on the user&apos;s device, and even though the code appears to work correctly when you are developing the app, it will fail in some locales. For example, in the Turkish locale, the uppercase replacement for `i` is **not** `I`.&#xA;&#xA;If you want the methods to just perform ASCII replacement, for example to convert an enum name, call `String#toUpperCase(Locale.US)` instead. If you really want to use the current locale, call `String#toUpperCase(Locale.getDefault())` instead."
-        url="http://developer.android.com/reference/java/util/Locale.html#default_locale"
-        urls="http://developer.android.com/reference/java/util/Locale.html#default_locale"
+        url="https://developer.android.com/reference/java/util/Locale.html#default_locale"
+        urls="https://developer.android.com/reference/java/util/Locale.html#default_locale"
         errorLine1="        override fun getItemViewType(position: Int): Int = ViewType.valueOf(hints[position].toUpperCase()).ordinal"
         errorLine2="                                                                                            ~~~~~~~~~~~">
         <location
-            file="/home/dsvoronin/Work/avito-android-github/samples/test-app/src/main/kotlin/com/avito/android/ui/RecyclerAsLayoutActivity.kt"
+            file="/home/dsvoronin/Work/avito-android-github/subprojects/android-test/ui-testing-core-app/src/main/kotlin/com/avito/android/ui/RecyclerAsLayoutActivity.kt"
```

Comment:
We could make it in a test project for a more stable output

Reply:
i agree, we also need unit test fixtures generation